.github/workflows/build-pr.yml

```diff
@@ -50,7 +50,7 @@ jobs:
         run: yarn jest --maxWorkers=2 --ci
 
       - name: Codecov
-        uses: codecov/codecov-action@29386c70ef20e286228c72b668a06fd0e8399192 # renovate: tag=v1.5.2
+        uses: codecov/codecov-action@c585afe366f940d214dc09df2664c06d7fe07052 # renovate: tag=v2.0.1
```

Comment:
I searched for `codecov` in the code, and checked the `codecov.yml` config file and our 2 workflow files which use `codecov/codecov-action` and did not find any properties that the changelog states are removed/deprecated.

I think this should merge without problems, but I am not 100% certain.

I'll let the Renovate team do their review now. 😉